packages/web/.gitignore

```diff
@@ -24,6 +24,9 @@ env/.env.dev.local
 /build-production
 /build-mobile-production
 /build-ssr
+/build-ssr-development
+/build-ssr-staging
+/build-ssr-production
```

Comment:
are these intended? assume you just included as a nice-to-have?

Reply:
Yes, didn't want to make a separate pr 🙈